.github/workflows/test.yml

```diff
@@ -30,14 +30,10 @@ jobs:
           retry_wait_seconds: 60
           max_attempts: 3
           command: npm i -g firebase-tools
-      - name: Use extension commands
-        run: firebase --open-sesame extdev
-      - name: Run 'firestore-send-email' extension emulator
+      - name: Run 'firestore-send-email' tests on the Extension emulator
         run:
-          cd ./firestore-send-email/functions/__tests__/emulator && sh
-          ./start.sh
-      - name: Run tests with coverage
-        run: npm run test-coverage
+          cd ./firestore-send-email/integration-test/ && firebase emulators:exec
+          "cd ../../ && npm run test-coverage" -P demo-test
```

Comment:
We could potentially remove this from the CI config and keep it local the extension. If the tests are ran locally, currently there is no emulator and the tests will fail.